extensions/cornerstone/src/OHIFCornerstoneViewport.js

```diff
@@ -139,9 +143,11 @@ class OHIFCornerstoneViewport extends Component {
     }
 
     this.getViewportData(displaySet).then(viewportData => {
-      this.setState({
-        viewportData,
-      });
+      if (viewportData) {
+        this.setState({
+          viewportData,
+        });
+      }
```

Comment:
Same here. These feels like protecting against something that shouldn't happen? If it happens, that means we're hiding a mistake we made somewhere else? What am I missing 🤔

Reply:
@igoroctaviano  realised It was being hit sometimes, so I put in these guards. We should check this more vigorously later.

Now you mention it, its strange as we aren't even using this viewport, we are using the measurement tracking ones. This did fix the error messages though, so strange its being hit?

Reply:
Lets try and review this seperate issue some point today.